deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euo pipefail
+
+# export KUBE_NAMESPACE=${ENVIRONMENT:-$DRONE_DEPLOY_TO}
+export KUBE_NAMESPACE="hocs-delta"
```

Comment:
Change this so that it's non-namespace specific; the commented line should be fine